libbeat/esleg/eslegclient/bulkapi.go

```diff
@@ -15,21 +15,39 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package elasticsearch
+package eslegclient
 
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"strings"
 
 	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/logp"
```

Comment:
This line addition should go away once https://github.com/elastic/beats/pull/17211 is merged (and this PR here is rebased on `7.x`).